open-source-services.md

```diff
@@ -28,19 +28,47 @@ Thus we must address common questions about the requirements:
 
 ### Q: What defines a service?
 
-A service is software operated for the user. Software the user is operating for themself is not considered a service. A service is defined by the context in which the software is operated and all of the assets required to operate the software for the user in that context.
+A service is software operated for the user.
+Software the user is operating for themself is not considered a service.
+A service is defined by the context in which the software is operated and all of the assets required to operate the software for the user in that context.
 
-### Q: What “assets” should be Open Source?
+This is a lightweight definition of service, read the Red Hat ["What are cloud services?"](https://www.redhat.com/en/topics/cloud-computing/what-are-cloud-services) article for a more broad discussion of cloud services.
```

Comment:
Should we wrap the word service in quotes? Otherwise this reads oddly.

Reply:
The challenge is, in English the use of "quote marks" can mean a lack of confidence about the words. But you are correct that we do want to emphasize it, which is typically done with italics. I'll make that suggestion into this one.